src/segmented-control/src/SegmentedControlRadio.js

```diff
@@ -53,7 +49,7 @@ const SegmentedControlRadio = memo(
           cursor="pointer"
           htmlFor={id}
           fontWeight={500}
-          size={textSize}
```

Comment:
Breaking change? Should we start cataloging everything that is breaking in a .md file?